dev-tools/cmd/dashboards/export_dashboards.go

```diff
@@ -68,7 +68,7 @@ func Export(client *http.Client, conn string, dashboard string, out string) erro
 	params.Add("dashboard", dashboard)
 
 	fullURL := makeURL(conn, exportAPI, params)
-	fmt.Printf("Calling HTTP GET %v\n", fullURL)
+	if ! quiet { fmt.Printf("Calling HTTP GET %v\n", fullURL) }
```

Comment:
Could you please add new lines after `{` and `fullURL)`?

Most of the codebase does not use this format. So in order to keep the uniformity of the code, please change it.